server/gdal_tools.py

```diff
@@ -0,0 +1,374 @@
+    Returns:
+        String containing the gdalinfo output or error message
+    """
+    if not validate_file_path(dataset):
+        return f"Error: File not found or not readable: {dataset}"
+    
+    cmd = ["gdalinfo"]
+    
+    if json_output:
+        cmd.append("-json")
+    
+    if stats:
+        cmd.append("-stats")
+    
+    cmd.append(dataset)
+    
+    result = run_gdal_command(cmd)
+    
+    if result["success"]:
+        return result["stdout"]
+    else:
+        return f"Error: {result['error']}"
+
+
+@mcp.tool()
+def gdal_translate(
+    src_dataset: str,
+    dst_dataset: str = "",
+    output_format: str = "GTiff",
+    bands: List[int] = None,
+    scale_min: float = None,
+    scale_max: float = None,
+    output_type: str = None
+) -> str:
+    """Convert raster data between formats using gdal_translate.
+    
+    This tool converts raster data between different formats and can also:
+    - Subset specific bands
+    - Rescale pixel values
+    - Change output data type
+    - Compress output files
+    
+    Args:
+        src_dataset: Path to source raster dataset
+        dst_dataset: Path to output dataset (auto-generated if empty)
+        output_format: GDAL output format (default: GTiff)
+        bands: List of band numbers to copy (1-based indexing)
+        scale_min: Minimum value for scaling
+        scale_max: Maximum value for scaling
+        output_type: Output data type (Byte, UInt16, Int16, UInt32, Int32, Float32, Float64)
+        
+    Returns:
+        String containing the result path or error message
+    """
+    if not validate_file_path(src_dataset):
+        return f"Error: Source file not found or not readable: {src_dataset}"
+    
+    # Generate output path if not provided
+    if not dst_dataset:
+        format_extensions = {
+            "GTiff": ".tif",
+            "JPEG": ".jpg",
+            "PNG": ".png",
+            "HFA": ".img",
+            "ENVI": ".dat"
+        }
+        ext = format_extensions.get(output_format, ".tif")
+        dst_dataset = generate_output_path(src_dataset, "_converted", ext)
+    
+    cmd = ["gdal_translate"]
+    
+    # Add format option
+    cmd.extend(["-of", output_format])
+    
+    # Add band selection
+    if bands:
+        for band in bands:
+            cmd.extend(["-b", str(band)])
+    
+    # Add scaling options
+    if scale_min is not None and scale_max is not None:
+        cmd.extend(["-scale", str(scale_min), str(scale_max)])
+    
+    # Add output type
+    if output_type:
+        cmd.extend(["-ot", output_type])
+    
+    # Add source and destination
+    cmd.extend([src_dataset, dst_dataset])
+    
+    result = run_gdal_command(cmd, timeout=120)
+    
+    if result["success"]:
+        return f"Successfully converted to: {dst_dataset}"
+    else:
+        return f"Error: {result['error']}"
+
+
+@mcp.tool()
+def gdalwarp(
+    src_datasets: List[str],
+    dst_dataset: str = "",
+    target_srs: str = "EPSG:4326",
```

Comment:
We should consider just using the epsg code as the arg, then handling the string concatenation under-the-hood.

CRS info can be represented in a lot of ways, and a string arg could create ambiguities here, whereas an epsg code is explicit:
 
```suggestion
    target_srs: int = 4326,
```